module/Photo/src/Photo/Mapper/Album.php

```diff
@@ -151,6 +151,61 @@ public function getRootAlbums()
     }
 
     /**
+     * Gets all root albums containing photos made between the specified dates
```

Comment:
Het klopt nog steeds niet. Nu geeft de functie alleen maar albums terug die alleen fotos in de time range hebben.

Als je een album van 1-6-2015 tot 1-8-2015 hebt, dan valt nu niet onder 2014 of 2015